tests/BlockchainTestsRunner.js

```diff
@@ -91,6 +91,9 @@ module.exports = function runBlockchainTest (options, testData, t, cb) {
             cb(err)
           })
         } catch (err) {
+          if (err) {
+            console.log(err)
```

Comment:
Did you encounter errors here when testing? In that case, maybe explicitly failing with `t.fail(err)` would help find the errors.

Reply:
Update: using `t.fail` will cause some invalid test cases to fail. We can just remove the `console.log` here.

Reply:
Good call! I updated the PR.

Reply:
Ah you changed it to `t.fail` :) sorry, my first comment was mistaken, I think we can ignore the error here, no need for any handling.

Reply:
I kinda like having the test fail here when it has err, otherwise it fails silently and make it a tad bit more difficult to debug. What do you think @s1na ?

Reply:
Sorry, my comment was meant for the other one (GeneralStateTestsRunner), see below.

Reply:
updated below